### PR TITLE
[PRG-2023] Add date and location, plus minor update

### PR DESCRIPTION
**Pull request by EtssMM**
*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2019] Add Bluth Company as a sponsor`*

If you are adding or removing organisers, please email info@devopsdays.org with the details of who is being added and removed, along with the full names and email addresses of the new team, so we can update Slack and the mailing list.

### CODE REVIEW

File: data/events/2023-prague.yml [hunk 1]
```diff
@@ -28,14 +28,14 @@ registration_link: "" # If you have a custom registration link, enter it here. T
 
 # Location
 #
-coordinates: "" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
+coordinates: "50.109730, 14.501550" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
 location: "prague" # Defaults to city, but you can make it the venue name.
```
Comment (phrawzty) [inline data/events/2023-prague.yml:32]: Not a blocker, but you'll probably want to modify this string in a future PR (if only to capitalise the place name, for example).

**Review by phrawzty — COMMENTED** [review]
Not a blocker, but you may wish to comment out the `sponsor` menu item in a future PR until you're ready to accept sponsors.

File: data/events/2023-prague.yml (hunk 1 above)
Comment (EtssMM) [reply data/events/2023-prague.yml:32]: good point, will update next time (this week)

**Comment by EtssMM** [comment]
The sponsor menu was left there on purpose. We are happy to start discussing the options with anyone who will email us even without the proposed levels being online. Maybe not the best practice but that was the intention. Anyway, the sponsors page will be updated in the next few days.

**Pull request merged**